src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -435,6 +435,10 @@ static inline ucs_status_t uct_dc_mlx5_iface_dci_get(uct_dc_mlx5_iface_t *iface,
 
     ucs_assert(!iface->super.super.config.tx_moderation);
 
+    if (!uct_dc_mlx5_iface_has_tx_resources(iface)) {
+        goto out_no_res;
```

Comment:
seems same as line 448?
should we count stats here?

Reply:
no, below we check for DCI resources by `uct_dc_mlx5_iface_dci_has_tx_resources`
will add stats